pom.xml

```diff
@@ -371,7 +371,7 @@
                             <configLocation>checkstyle.xml</configLocation>
                             <includeTestSourceDirectory>true</includeTestSourceDirectory>
                             <encoding>UTF-8</encoding>
-                            <consoleOutput>true</consoleOutput>
+                            <consoleOutput>false</consoleOutput>
                             <failsOnError>false</failsOnError>
```

Comment:
What is the reason to have Checkstyle in the project if severity set to `warning` and  console output is disabled?